pkgs/servers/openafs/cellservdb.nix

```diff
@@ -0,0 +1,23 @@
+{
+  fetchurl,
+  lib,
+}:
+let
+  version = "2023-10-31";
```

Comment:
FWIW, https://grand.central.org/dl/cellservdb/CellServDB.2025-08-16 is now out. (See also https://lists.central.org/pipermail/cellservdb-announce/2025/000046.html)

(Also http://dl.central.org/ -- as opposed to https:// -- seems to be down at the moment. I've sent an email; I assume that's not expected to be true long-term. Also FWIW the announcement email and https://grand.central.org/csdb.html reference "grand.central.org", not "dl.central.org", although they also don't reference specific versions -- I don't know if dl is preferred over grand or vice-versa for our usecase.)